Source/Resources/Content/MSGraph/DeviceManagement/DeviceCompliancePolicies/Corporate Devices - Physical - Windows.json

```diff
@@ -15,7 +15,7 @@
   "codeIntegrityEnabled": false,
   "configurationManagerComplianceRequired": false,
   "defenderEnabled": true,
-  "description": "End User device compliance",
+  "description": "What is it: Defines the required state that a physical device must be in to be considered compliant and before it can access tenant data. This policy applies only to physical devices since some requirements such as BitLocker are not supported on virtual devices. \n Simeon Notes: Provides an additional layer of data protection for your Intune managed devices. Simeon's baseline requires that a physical device has a number of security controls configured including BitLocker encryption turned on and a Microsoft Defender ATP risk score of Medium or less. Assignment is based on the Azure AD group 'BUILTIN - Corporate Devices – Physical'. \n Learn More: https://docs.microsoft.com/en-us/mem/intune/protect/device-compliance-get-started",
```

Comment:
**What does this do?**
Defines the required state that a physical device must be in to be considered compliant and before it can access tenant data. The baseline requires that a physical device has a number of security controls configured including BitLocker encryption and a Microsoft Defender ATP risk score of Medium or less. Assignment is based on the Azure AD group 'Baseline - Corporate Devices – Physical'. This policy applies only to physical devices since some requirements such as BitLocker are not supported on virtual devices. 

**Why should you use this?**
This provides an additional layer of data protection for your Intune managed devices. 

**What is the end-user impact?**
Users may only access tenant data if BitLocker encryption is enabled and the Microsoft Defender ATP risk score is Medium or less. 

@metamenos should "medium" be capitalized? Also, is organizational data the same thing as tenant data? if so we need to choose one to be consistent

**Learn more**
[Use compliance policies to set rules for devices you manage with Intune](https://docs.microsoft.com/en-us/mem/intune/protect/device-compliance-get-started)

Reply:
no medium doesn't need caps, and yes tenant is same as org.  So yeah good to be consistent